supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -608,10 +615,7 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     val outputs = call.task.outputs
     val outputMappings = outputs.foldLeft(Seq.empty[AttemptedLookupResult])(outputFoldingFunction).map(_.toPair).toMap
     TryUtil.sequenceMap(outputMappings) map { outputMap =>
-      outputMap mapValues { v =>
-        // PBE fix hashing
-        JobOutput(v, hash = None)
-      }
+      outputMap mapValues { v => JobOutput(v) }
```

Comment:
I think we know we need call caching, so a non-TODO comment in code isn't too useful IMO.

Also, the engine isn't the one doing call caching so sending the hash back to the engine isn't the place to do call caching in any case.